docs/try-snowplow/setting-up-your-pipeline/installing-try-snowplow-on-aws/index.md

```diff
@@ -22,7 +22,7 @@ Snowplow will never ask you to disclose your Amazon, Google or database credenti
 
 Setting up Try Snowplow in your AWS account requires following IAM roles:
 
-```
+```bash
```

Comment:
why bash?

Reply:
I normally go for `text` for non-code snippets, it won't be colourful (which is fine imo) but more accurate, I feel.

Reply:
Adjusted to text (which isn't in Prism list - https://prismjs.com/#supported-languages), so gives the no colour/no format.

Yes, a better option :)